source/tutorials/moving-pictures.rst

```diff
@@ -59,6 +59,12 @@ The semantic type of this QIIME 2 artifact is ``EMPSingleEndSequences``. ``EMPSi
      --input-path emp-single-end-sequences \
      --output-path emp-single-end-sequences.qza
 
+It is possible to check the ID, type, and format of your newly-imported sequences, confirming that your import worked as expected:
+
+.. command-block::
+
```

Comment:
This would be a great place to include a :stdout: flag, for example, see this command block:

https://github.com/qiime2/docs/blame/master/source/tutorials/utilities.rst#L99-L102

And here it is rendered:

https://docs.qiime2.org/2020.2/tutorials/utilities/#peeking-at-results

Reply:
Good call. Thanks!